README.md

```diff
@@ -1,128 +1,209 @@
+Hermione is the utility for integration testing of web pages using [WebdriverIO](http://webdriver.io/) and [Mocha](https://mochajs.org).
 
-## Установка
+<img src="hermione.png" align="right"/>
 
-```shell
-npm i hermione --save-dev
-```
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
 
-## Запуск тестов
 
-```shell
-node_modules/.bin/hermione path/to/tests --baseUrl http://yandex.ru/search --grid http://localhost:4444/wd/hub
-```
+- [Why should you choose hermione?](#why-should-you-choose-hermione)
+  - [Easy to use](#easy-to-use)
+  - [Parallel test running](#parallel-test-running)
+  - [Extensibility](#extensibility)
+  - [Retryies for failed tests](#retryies-for-failed-tests)
+  - [Executing a separate test](#executing-a-separate-test)
+  - [Auto initilization and closing grid sessions](#auto-initilization-and-closing-grid-sessions)
+- [Prerequisites](#prerequisites)
+- [Quick start](#quick-start)
+- [.hermione.conf.js](#hermioneconfjs)
+  - [specs](#specs)
+  - [browsers](#browsers)
+  - [grid](#grid)
+  - [baseUrl](#baseurl)
+  - [timeout](#timeout)
+  - [waitTimeout](#waittimeout)
+  - [slow](#slow)
+  - [debug](#debug)
+  - [sessionsPerBrowser](#sessionsperbrowser)
+  - [retry](#retry)
+  - [plugins](#plugins)
+  - [mochaOpts](#mochaopts)
+  - [prepareBrowser](#preparebrowser)
+  - [prepareEnvironment](#prepareenvironment)
+- [CLI](#cli)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Why should you choose hermione?
+`Hermione` provides several features which `WebdriverIO` doesn't and makes testing process easier.
+
+### Easy to use
+If you know `Mocha` and `WebdriverIO`, you can start writting and running tests in 5 minutes! You need to install `hermione` via npm and add a tiny config in your project. See details in [Quick start](#quick-start) section.
+
+### Parallel test running
+When tests are run one by one, it'll cost a lot of time. `Hermione` can run tests in parallel sessions in different browsers out of the box.
+
+### Extensibility
+`WebdriverIO` provides built-in commands for browser and page manipulation. Often projects are needed to store some common code and reuse it though all tests. So a developer should create some helpers and include them in tests.
+
+With `hermione` is very simple and straightforward. You can add any number of custom commands in the hermione config and use them as `this.browser.myCustomCommand` in tests.
+
+Moreover, `hermione` provides plugins which works as a some kind of hooks. They allow a developer to prepare environement for tests and react properly to test execution events.
 
-## Конфигурация
+### Retryies for failed tests
+Integration tests use a dynamic environment with a lot of dependencies where any of them can work unstable from time to time. As a result, integration tests become red randomly and make them undetermined. It spoils all testing process.
```

Comment:
`randomly` -> `sometimes`. seems more appropriate in this context.
`tests sometimes fail randomly` will be enough though.